tests/kuttl/test-suites/default/watcher-decision-engine/03-deploy-decision-engine.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: watcher.openstack.org/v1beta1
+kind: WatcherDecisionEngine
+metadata:
+  name: watcher-kuttl
```

Comment:
To make it more clear I'd call the WatcherDecisionEngine CR, watcher-decision-engine-kuttl.

Reply:
ack